netebpfext/net_ebpf_ext_xdp.c

```diff
@@ -321,6 +334,10 @@ _net_ebpf_ext_allocate_cloned_nbl(_Inout_ net_ebpf_xdp_md_t* net_xdp_ctx, uint32
         IoFreeMdl(mdl_chain);
     if (packet_buffer != NULL)
         ExFreePool(packet_buffer);
+
+    if (!NT_SUCCESS(status))
+        NET_EBPF_EXT_LOG_FUNCTION_ERROR(status);
+
     return status;
```

Comment:
```suggestion
    NET_EBPF_EXT_RETURN_RESULT(status);
```

Reply:
`NET_EBPF_EXT_RETURN_RESULT` logs both success and failure. Since `_net_ebpf_ext_allocate_cloned_nbl` is a high frequency function, I just wanted to log the failures in this case.